config/hscConfig.py

```diff
@@ -15,17 +13,6 @@
 config.processCcd.calibrate.astromRefObjLoader.retarget(LoadAstrometryNetObjectsTask)
 config.processCcd.calibrate.photoRefObjLoader.retarget(LoadAstrometryNetObjectsTask)
```

Comment:
Do we also need to take out the references to `LoadAstrometryNetObjectsTask`?

Reply:
ah perhaps not .. indeed if it needs to go then I missed it in retarget. So perhaps I should indeed return these

Reply:
indeed tim says meas_extensions_astrometryNet was removed from the stack last year some time